documentation/CONTRIBUTING.md

```diff
@@ -72,16 +72,33 @@ default values aren't appropriate:
   * `PDB_TEST_DB_ADMIN` (defaults to `pdb_test_admin`)
   * `PDB_TEST_DB_ADMIN_PASSWORD` (defaults to `pdb_test_admin`)
 
-Then you can run the test suite:
+Then you can run the unit tests:
 
     $ lein test
 
 And if you'd like to preserve the temporary test databases on failure, you can
 set `PDB_TEST_PRESERVE_DB_ON_FAIL` to true:
 
-```
-$ PDB_TEST_KEEP_DB_ON_FAIL=true lein test
-```
+    $ PDB_TEST_KEEP_DB_ON_FAIL=true lein test
+
+To run the integration tests, you'll need to ensure you have a
+suitable verion of Ruby available, and then install the relevant gems
+by running something like this from the top level of the source tree:
+
+    $ bundle install --path ~/.gem
+
+`bundle` will store the gems in `~/.gem` and will remember that
+location for future invocations.  You're welcome to choose a different
+`--path`.
```

Comment:
@wkalt After talking to @mullr I'm changing this to recommend `--path "$(pwd)/.vendor" for now, since the integration tests mangle some of the gems, but Ruth may help us improve/avoid that.